client/app/states/requests/details/details.state.js

```diff
@@ -32,10 +32,60 @@
   }
 
   /** @ngInject */
-  function RequestDetailsController(request) {
+  function RequestDetailsController(request, CollectionsApi, DialogFieldRefresh, EventNotifications) {
     var vm = this;
+    vm.editingDisabled = true;
 
     vm.title = request.description;
     vm.request = request;
+    vm.dialogs = [];
+    vm.dialogs.push(request.provision_dialog);
+
+    var autoRefreshableDialogFields = [];
+    var allDialogFields = [];
+
+    DialogFieldRefresh.setupDialogData(vm.dialogs, allDialogFields, autoRefreshableDialogFields);
+
+    var dialogUrl = 'service_catalogs/' + vm.request.source_id + '/service_templates';
+
+    angular.forEach(allDialogFields, function(dialogField) {
+      dialogField.refreshSingleDialogField = function() {
+        DialogFieldRefresh.refreshSingleDialogField(allDialogFields, dialogField, dialogUrl, vm.request.source_id);
+      };
+    });
+
+    function saveRequest() {
+      var dialogData = dataForSubmit();
+      var data = { action: 'edit', dialogData };
+      CollectionsApi.post(
+        'service_requests',
+        request.id,
+        {},
+        { action: 'edit', options: { dialog: dialogData } }
```

Comment:
BTW, this ticket is most definitely a work in progress and is not ready for review.  I am committing to it merely as a backup for my laptop. Thanks